packages/transaction-controller/src/TransactionController.ts

```diff
@@ -776,23 +776,48 @@ export class TransactionController extends BaseController<
             value: '0x0',
           };
 
-    const unsignedEthTx = this.prepareUnsignedEthTx(txParams);
+    const unsignedEthTx = this.prepareUnsignedEthTx(cancelTxParams);
 
     const signedTx = await this.sign(
       unsignedEthTx,
       transactionMeta.txParams.from,
     );
     await this.updateTransactionMetaRSV(transactionMeta, signedTx);
```

Comment:
This may have been accidental at the time as I can't see why we'd want to update the signature properties on the existing transaction since that is unchanged?

Reply:
I have no good reason to answer this, seems like removing is the best action that we take here https://github.com/MetaMask/core/pull/1998/commits/c5072ed8b5be1a4d85d431dd590a6810ed75940b